build/ci/support/cleanup.jenkinsfile

```diff
@@ -22,7 +21,8 @@ pipeline {
                 retry(3)
             }
             steps {
-                sh 'make -C build/ci ci-gke-cleanup'
+                createRunConfig('delete', "${GKE_CLUSTER}")
+                sh 'make -C build/ci ci-run-deployer'
```

Comment:
Could we just invoke this directly in the job? Job name is general and this will only work for gke.

Reply:
Fixed